roles/taca/templates/site_taca_sarek_delivery.yml.j2

```diff
@@ -0,0 +1,61 @@
+log:
+    file: "{{ ngi_pipeline_site_path }}/log/taca.log"
+
+deliver:
+    rootpath: "{{ proj_root }}/{{ ngi_pipeline_site_delivery }}/nobackup/NGI"
+    analysispath: <ROOTPATH>/ANALYSIS/<PROJECTID>
+    stagingpath: <ROOTPATH>/DELIVERY/<PROJECTID>
+    operator: "{{ recipient_mail }}"
+    hash_algorithm: md5
+    files_to_deliver:
+{% if "upps" == site %}
+        -
+            - <ANALYSISPATH>/seqreports/*
+            - <STAGINGPATH>/00-Reports/SequenceQC/
+            - required: True
+        -
+            - <ANALYSISPATH>/multiqc_ngi/*multiqc_report.html
+            - <STAGINGPATH>/00-Reports/
+            - required: True
+        -
+            - <ANALYSISPATH>/multiqc_ngi/*multiqc_report_data.zip
+            - <STAGINGPATH>/00-Reports/
+            - required: True
+{% endif %}
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Annotation/*
+            - <STAGINGPATH>/<SAMPLEID>/results/Annotation/
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/pipeline_info/results_description.html
+            - <STAGINGPATH>/<SAMPLEID>/results/pipeline_info
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/pipeline_info/software_versions.csv
+            - <STAGINGPATH>/<SAMPLEID>/results/pipeline_info
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Preprocessing/TSV/duplicates_marked*
+            - <STAGINGPATH>/<SAMPLEID>/results/Preprocessing/TSV
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Preprocessing/<SAMPLEID>/DuplicatesMarked/*
+            - <STAGINGPATH>/<SAMPLEID>/results/Preprocessing/DuplicatesMarked
+            - required: True
+        -
+            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Reports/*
```

Comment:
```suggestion
            - <ANALYSISPATH>/<SAMPLEID>/SarekGermlineAnalysis/results/Reports/<SAMPLEID>/*
```

Reply:
I thought it would be nice to include report files for the "lane bams" since it is included in the MultiQC that is made for the whole project. Also, I don't see why we should remove the MultiQC reports on sample level? They are a bit unnecessary since it is basically the same info as in the project report, but IMO we should alter the sarek output as little as possible. Let me know what you think.

Reply:
ok, I realize that I'm thinking of the more general case where multiple samples have been analyzed together. In that case the reports folder will hold multiple samples. that's not how we run the analysis though and that's of course reflected in the first part of the path... my bad, all is well!